### PR TITLE
feat(PW-110): CMS csr collection

### PR DESCRIPTION
**Pull request by alimarf**
added query

### CODE REVIEW

File: sanity/services/csr-query.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const csrQuery = `*[_type == "csr"]{
+    _id,
+    title,
+    titleColor,
+    subtitle,
+    description,
+    images,
```
Comment (bimaahida) [inline sanity/services/csr-query.ts:7]: which we use for generate the url image ?
using groq or using helper in code ?

please standarize for that

File: sanity/schemaTypes/csr.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            options: {
+                layout: "grid",
+            },
+            validation: (Rule) => Rule.required(),
+        }),
+        defineField({
+            name: "position",
+            title: "Position",
+            type: "number",
+            validation: (Rule) => Rule.required(),
```
Comment (bimaahida) [inline sanity/schemaTypes/csr.ts:54]: please add validation for duplicate position

**Review by bimaahida — COMMENTED** [review]
we have some comment, please take a look for that
Thanks

File: sanity/schemaTypes/csr.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    ],
+    preview: {
+        select: {
+            title: "title",
+            subtitle: "subtitle",
+            media: "images",
+        },
+        prepare(selection) {
+            const { title, subtitle, media } = selection;
+            const firstImage = media.find((m: any) => m._type === "image");
```
Comment (bimaahida) [inline sanity/schemaTypes/csr.ts:65]: why we need filter for type image, since we only have 1 type

Comment (alimarf) [reply sanity/schemaTypes/csr.ts:65]: Okey i'll change that one

File: sanity/schemaTypes/csr.ts (hunk 2 above)
Comment (alimarf) [reply sanity/schemaTypes/csr.ts:54]: roger that!

File: sanity/services/csr-query.ts (hunk 1 above)
Comment (alimarf) [reply sanity/services/csr-query.ts:7]: Sure

File: sanity/schemaTypes/csr.ts [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+                            if (isUnique) {
+                                return true;
+                            }
+
+                            throw new Error("Position must be unique");
+                        } catch (error: any) {
+                            return error.message;
+                        }
+                    }
```
Comment (ilyas-muhammad) [inline sanity/schemaTypes/csr.ts:80]: if we make this function reusable - that'll be great

**Review by ilyas-muhammad — APPROVED** [review]

**Review by ilyas-muhammad — APPROVED** [review]

**Pull request merged**